### DISCUSSION TITLE
执行两个大表的join操作时集群挂掉并且be下产生大量core.*文件

### DISCUSSION THREAD

**Discussion by LCehoennardo** [apache/doris#12190] [A - General / Q&A]
执行两个大表的join操作时集群的be直接挂掉，并且会在be目录下产生大量的core.*文件，我们排查了be和fe的日志，没有找到原因，`dmesg -T`也没看到什么信息，请问可能是什么原因呢？这些core.*文件是什么？
<img width="465" alt="image" src="https://user-images.githubusercontent.com/41560874/187429808-b22cb93e-d784-4f92-b995-37042e97a644.png">

**Comment by LCehoennardo** [comment]
对了，doris 版本是1.0.0，而且这个问题是必现的

**Reply by dataalive** [reply]
这个应该是触发了某个bug导致的BE CORE，可以联系我们看看错误栈，或者贴下be.out里的栈。建议升级到即将发布的1.1.2版本，修复了很多对应的稳定性问题。

**Reply by LCehoennardo** [reply]
好的 多谢 升级的问题我们得评估一下

**Comment by hf200012** [comment]
@LCehoennardo 你加我微信：35926237 ，我给你个1.1.2 的包你升级上去看看

**Reply by LCehoennardo** [reply]
谢谢 不用了 我的一位同事已经有1.1.2的包了
